### ISSUE TITLE
The accuracy on YFCC with GL3D model

### ISSUE THREAD

**Issue by guohengkai** [zjhthu/OANet#25]
Hi, I have a question about the GL3D model. When I test it on the YFCC test set, the results are very bad:
test result [0.09993750000000001, 0.029445680575755823, 0.5177923955358564, 0.9643927037450776, 0.446788042617176, 0.6026962917102501, 0.46986473430161074]

Is it something wrong with the test setting? And what hyperparameters do you use for training the GL3D model? I cannot see any codes in the repo.

Thank you very much.

**Comment by zjhthu** [comment]
The model trained on GL3D is a fundamental matrix estimation model, you should set `--use_fundamental=True`. 
Besides, it is reasonable the model trained on GL3D does not work well on YFCC, since these two datasets are very much different.

**Comment by guohengkai** [comment]
OK. Thank you for your response.
